packages/devtools_app/test/memory/control/settings_dialog_test.dart

```diff
@@ -17,52 +17,19 @@ import 'package:devtools_shared/devtools_shared.dart';
 import 'package:devtools_test/devtools_test.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:mockito/mockito.dart';
 
 import '../../matchers/matchers.dart';
 import '../../test_data/memory.dart';
 import '../../test_data/memory_allocation.dart';
 
 void main() {
   late MemoryController controller;
-  late FakeServiceManager fakeServiceManager;
-  late PreferencesController preferencesController;
 
-  void _setUpServiceManagerForMemory() {
-    // Load canned data testHeapSampleData.
-    final memoryJson =
-        SamplesMemoryJson.decode(argJsonString: testHeapSampleData);
-    final allocationJson =
-        AllocationMemoryJson.decode(argJsonString: testAllocationData);
-
-    fakeServiceManager = FakeServiceManager(
-      service: FakeServiceManager.createFakeService(
-        memoryData: memoryJson,
-        allocationData: allocationJson,
-      ),
-    );
-    when(fakeServiceManager.connectedApp!.isDartWebAppNow).thenReturn(false);
-    when(fakeServiceManager.connectedApp!.isFlutterAppNow).thenReturn(true);
-    when(fakeServiceManager.connectedApp!.isDartCliAppNow).thenReturn(false);
-    when(fakeServiceManager.connectedApp!.isDebugFlutterAppNow)
-        .thenReturn(false);
-    when(fakeServiceManager.connectedApp!.isDartWebApp)
-        .thenAnswer((_) => Future.value(false));
-    setGlobal(ServiceConnectionManager, fakeServiceManager);
-
-    controller.offline.value = true;
-    controller.memoryTimeline.offlineData.clear();
-    controller.memoryTimeline.offlineData.addAll(memoryJson.data);
-  }
-
-  Future<void> pumpMemoryScreen(
-    WidgetTester tester, {
-    MemoryController? memoryController,
-  }) async {
+  Future<void> pumpMemoryScreen(WidgetTester tester) async {
     await tester.pumpWidget(
       wrapWithControllers(
         const MemoryBody(),
-        memory: controller = memoryController ?? MemoryController(),
+        memory: controller = controller,
```

Comment:
it looks like we are just assigning `controller` to itself, so this shouldn't be necessary.